bolt-modules/boltlib/spec/functions/apply_prep_spec.rb

```diff
@@ -60,36 +60,18 @@
       Puppet::Pal::ScriptCompiler.any_instance.stubs(:task_signature).with('service').returns(task3)
     end
 
-    it 'sets feature and gathers facts' do
```

Comment:
Am I reading this right, did we literally just have the same test here twice?

Reply:
Yep